src/localization/fi.json

```
@@ -299,5 +299,31 @@
       "roomName" : "Huoneen nimi",
       "dialogTitle" : "Anna huoneen nimi"
     }
+  },
+
+  "layoutEditor": {
+    "commonComponents": {
+      "id": "ID",
+      "layoutWidth": "Layout width",
+      "layoutHeight": "Layout height",
+      "backgroundColor": "Layout background color",
+      "paddings": {
+        "left": "Padding left",
+        "right": "Padding right",
+        "top": "Padding top",
+        "bottom": "Padding bottom",
+        "link": "Link paddings"
+      },
+      "margins": {
+        "left": "Margin left",
+        "right": "Margin right",
+        "top": "Margin top",
+        "bottom": "Margin bottom",
+        "link": "Link margins"
+      },
+      "layoutGravity": "Layout gravity",
+      "layoutAlign": "Layout align"
+
+    }
```

A tad bit too much Finglish here.. maybe like this?
```suggestion
  },

  "layoutEditor": {
    "commonComponents": {
      "id": "ID",
      "layoutWidth": "Asettelun leveys",
      "layoutHeight": "Asettelun korkeus",
      "backgroundColor": "Asettelun taustaväri",
      "paddings": {
        "left": "Täyte vasemmalla",
        "right": "Täyte oikealla",
        "top": "Täyte yläpuolella",
        "bottom": "Täyte alapuolella",
        "link": "Linkkien täytteet"
      },
      "margins": {
        "left": "Marginaali vasemmalla",
        "right": "Marginaali oikealla",
        "top": "Marginaali yläpuolella",
        "bottom": "Marginaali alapuolella",
        "link": "Linkkien marginaalit"
      },
      "layoutGravity": "Asettelun painotus",
      "layoutAlign": "Asettelun tasaus"

    }
```